src/components/ui/drop-files/types.ts

```diff
@@ -42,6 +42,8 @@ export interface DropFilesTextsProp {
 }
 
 export interface DropFilesProps {
+  /** Arquivo inicial a ser carregado. */
+  file?: File | null;
```

Comment:
Não faria sentido a prop também poder ser um path para um arquivo e o componente se encarregaria de gerar o file internamente. Acho que isso facilitaria o uso, visto que a maioria dos locais que for ser iniciado teremos apenas o link ou caminho do arquivo, não diretamente em um `File`.

```suggestion
  /** Arquivo inicial a ser carregado. */
  file?: String | File | null;
```

Reply:
O padrão de outras libs é usar só o `File` como entrada e além disso por se tratar de um design-system é melhor deixarmos para controle de quem implementa o DS no seu projeto implementar a função de tratar o File, já que podemos acabar assumindo mais responsabilidades do deveriamos nesse caso.